modules/post/osx/manage/record_mic.rb

```diff
@@ -12,59 +12,60 @@ class MetasploitModule < Msf::Post
 
   POLL_TIMEOUT = 120
 
-  def initialize(info={})
-    super(update_info(info,
-      'Name'          => 'OSX Manage Record Microphone',
-      'Description'   => %q{
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'OSX Manage Record Microphone',
+        'Description' => %q{
           This module will allow the user to detect (with the LIST action) and
           capture (with the RECORD action) audio inputs on a remote OSX machine.
-      },
-      'License'       => MSF_LICENSE,
-      'Author'        => [ 'joev'],
-      'Platform'      => [ 'osx'],
-      'SessionTypes'  => [ 'shell' ],
-      'Actions'       => [
-        [ 'LIST',     { 'Description' => 'Show a list of microphones' } ],
-        [ 'RECORD', { 'Description' => 'Record from a selected audio input' } ]
-      ],
-      'DefaultAction' => 'LIST'
-    ))
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'joev'],
+        'Platform' => [ 'osx'],
+        'SessionTypes' => [ 'shell' ],
+        'Actions' => [
+          [ 'LIST', { 'Description' => 'Show a list of microphones' } ],
+          [ 'RECORD', { 'Description' => 'Record from a selected audio input' } ]
+        ],
+        'DefaultAction' => 'LIST'
+      )
+    )
 
     register_options(
       [
         OptInt.new('MIC_INDEX', [true, 'The index of the mic to use. `set ACTION LIST` to get a list.', 0]),
         OptString.new('TMP_FILE',
-          [true, 'The tmp file to use on the remote machine', '/tmp/.<random>/<random>']
-        ),
+                      [true, 'The tmp file to use on the remote machine', '/tmp/.<random>/<random>']),
         OptString.new('AUDIO_COMPRESSION',
-          [true, 'Compression type to use for audio', 'QTCompressionOptionsHighQualityAACAudio']
-        ),
+                      [true, 'Compression type to use for audio', 'QTCompressionOptionsHighQualityAACAudio']),
         OptInt.new('RECORD_LEN', [true, 'Number of seconds to record', 30]),
         OptInt.new('SYNC_WAIT', [true, 'Wait between syncing chunks of output', 5])
-      ])
+      ]
+    )
   end
 
-
   def run
-    fail_with(Failure::BadConfig, "Invalid session ID selected.") if client.nil?
-    fail_with(Failure::BadConfig, "Invalid action") if action.nil?
+    fail_with(Failure::BadConfig, 'Invalid session ID selected.') if client.nil?
+    fail_with(Failure::BadConfig, 'Invalid action') if action.nil?
 
-    num_chunks = (datastore['RECORD_LEN'].to_f/datastore['SYNC_WAIT'].to_f).ceil
-    tmp_file = datastore['TMP_FILE'].gsub('<random>') { Rex::Text.rand_text_alpha(10)+'1' }
+    num_chunks = (datastore['RECORD_LEN'].to_f / datastore['SYNC_WAIT']).ceil
```

Comment:
You left a comment above that removing `to_f` isn't always safe, is this definitely ok to do here?

Reply:
If the datastore is working as expected, this should be nornalized as an integer already:

```ruby
        OptInt.new('SYNC_WAIT', [true, 'Wait between syncing chunks of output', 5])
```

Reply:
I'm going to disable `Style/FloatDivision` - it's fine for new code, but too risky to run on all of Metasploit IMO